### DISCUSSION TITLE
Viewing NFS rewinds data

### DISCUSSION THREAD

**Discussion by shawnahall71** [NetApp/harvest#2430] [Q&A]
Hi,

Does Harvest collect NFS rewind statistics - e.g. the stats mentioned under [this support article](https://kb.netapp.com/onprem/ontap/da/NAS/High_Network_Processing_(CPU_N-blade)_latency_and_poor_performance_due_to_locking)?

That would be helpful in diagnosing a performance issue we're seeing.

**Comment by rahulguptajss** [comment]
@shawnahall71 Harvest doesn't collect these stats currently. Could you please open a feature request for this.

**Comment by shawnahall71** [comment]
Thanks - opened issue #2434

**Comment by rahulguptajss** [comment]
Thanks!
